Makefile

```diff
@@ -63,7 +63,8 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+	$(YARN) check
```

Comment:
Aha. I see. Does that not mean that the very first time a contributor clones the repo that `make setup` won't install the project dependencies?